src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        printLogo();
+        greetUser();
+
+        String userLineInput = "initialize";
+        Task[] taskItems = new Task[100]; //create array to store text entered by user
+        int taskCounter = 0; //track number of tasks
+
+        Scanner in = new Scanner(System.in); //Create scanner object to take in input
```

Comment:
Consider writing comments above the code, relative to the position of the code, as per coding standards.